test/root/RootChain.test.js

```diff
@@ -62,24 +64,43 @@ contract('RootChain', async function (accounts) {
     }
   })
 
+  it('try to submitHeaderBlock with negative vote(`00` prefix) and fail', async function () {
+    const validators = [1, 2, 3, 4]
+    let tree = await rewradsTree(validators, accountState)
+    const { data, sigs } = buildSubmitHeaderBlockPaylod(
+      accounts[0],
+      0,
+      255,
+      '' /* root */,
+      wallets,
+      { allValidators: true, rewardsRootHash: tree.getRoot(), getSigs: true, totalStake: totalStake, sigPrefix: '0x00' }
+    )
+    try {
+      await rootChain.submitHeaderBlock(data, sigs)
+    } catch (error) {
+      const invalidOpcode = error.message.search('revert') >= 0
```

Comment:
Can we assert on the actual expected reverted msg? Otherwise if it fails for some other reason the test won't detect it.

Reply:
revert comes from `ecrecover` so no error msg. I think we can add some msg there and assert the same here.